packages/rum/src/boot/recorderApi.spec.ts

```diff
@@ -135,6 +135,13 @@ describe('makeRecorderApi', () => {
       expect(startRecordingSpy).not.toHaveBeenCalled()
     })
 
+    it('should start recording if session is tracked without session replay when forced', () => {
+      setupBuilder.withSessionManager(createRumSessionManagerMock().setTrackedWithoutSessionReplay()).build()
+      rumInit()
+      recorderApi.start({ force: true })
+      expect(startRecordingSpy).toHaveBeenCalledTimes(1)
```

Comment:
**💬 suggestion:** ‏add an assertion to make sure `setForcedReplay` has been called.

Reply:
Done ✔️